tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -282,10 +276,7 @@ def test_monitoring_when_osd_down(self, pods):
             )
 
         # Make osd up which was down
-        params = '{"spec": {"replicas": 1}}'
-        assert ocp_obj.patch(resource_name=resource_name, params=params), (
-            f"Failed to change the replica count of osd {resource_name} to 1"
-        )
+        assert modify_osd_replica_count(resource_name=resource_name, replica_count=1)
```

Comment:
have you checked after bringing ods back data is still there and consuming the same pvc which was attached before

Reply:
No I haven't checked. This test case to check whether osd down will impact the functionality of monitoring backed by ocs so didn't check after bringing osd back data is still there.

It is good to have a separate test case to validate this condition. What's your thought on it?